sklearn/impute/_iterative.py

```diff
@@ -81,45 +81,46 @@ class IterativeImputer(_BaseImputer):
         imputation process, the neighbor features are not necessarily nearest,
         but are drawn with probability proportional to correlation for each
         imputed target feature. Can provide significant speed-up when the
-        number of features is huge. If ``None``, all features will be used.
+        number of features is huge. If `None`, all features will be used.
 
-    initial_strategy : str, default='mean'
+    initial_strategy : {'mean', 'median', 'most_frequent', 'constant'}, \
+            default='mean'
         Which strategy to use to initialize the missing values. Same as the
-        ``strategy`` parameter in :class:`~sklearn.impute.SimpleImputer`
-        Valid values: {"mean", "median", "most_frequent", or "constant"}.
+        `strategy` parameter in :class:`~sklearn.impute.SimpleImputer`.
 
-    imputation_order : str, default='ascending'
+    imputation_order : {'ascending', 'descending', 'roman', 'arabic', \
+            'random'}, default='ascending'
         The order in which the features will be imputed. Possible values:
 
-        "ascending"
+        - `'ascending'`
```

Comment:
In the rendering of the doc, I think that we can put the option and the description on the same line. Here and for the subsequent available options for this parameter.